metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -102,8 +104,8 @@ func (sink *StackdriverSink) Stop() {
 }
 
 func (sink *StackdriverSink) processMetrics(metricValues map[string]core.MetricValue,
-	timestamp time.Time, labels map[string]string, collectionStartTime time.Time, entityCreateTime time.Time) []*sd_api.TimeSeries {
-	timeseries := make([]*sd_api.TimeSeries, 0)
+	timestamp time.Time, labels map[string]string, collectionStartTime time.Time, entityCreateTime time.Time) []*monitoringpb.TimeSeries {
+	timeseries := make([]*monitoringpb.TimeSeries, 0)
```

Comment:
`var timeseries []*monitoringpb.TimeSeries`? That way empty arrays don't allocate.

Reply:
Done